src/app/messages/en/messages.json

```diff
@@ -75,8 +75,10 @@
   "i18n:notification:update:reload": "Update",
   "i18n:notification:search-failure": "We're unable to find that search result. Please try another.",
   "i18n:nudge:study-tools:heading": "The study tools you need. <strong>100% FREE!</strong>",
-  "i18n:nudge:study-tools:text": "Boost your studies with expert-created study guides and highlighting. <strong>It's all free.</strong>",
-  "i18n:nudge:study-tools:aria-label": "Image points to the Study Guides and My Highlights buttons in the toolbar. The study tools you need. 100% free. Boost your studies with expert-created study guides and highlighting. To close image, navigate to or click the x button.",
+  "i18n:nudge:study-tools:text:with-study-guides": "Boost your studies with expert-created study guides and highlighting. <strong>It's all free.</strong>",
+  "i18n:nudge:study-tools:text:only-highlighting": "Highlight, take notes, and make your own study guides. <strong>It's all free.</strong>",
+  "i18n:nudge:study-tools:aria-label:with-study-guides": "Image points to the Study Guides and My Highlights buttons in the toolbar. The study tools you need. 100% free. Boost your studies with expert-created study guides and highlighting. To close image, navigate to or click the x button.",
+  "i18n:nudge:study-tools:aria-label:only-highlighting": "Image points to the My Highlights buttons in the toolbar. The study tools you need. 100% free. Highlight, take notes, and make your own study guides. To close image, navigate to or click the x button.",
```

Comment:
The text is different from design doc, but this seems more accurate. Was the doc not updated? Also shouldn't it be  `My highlights button`?